src/app/core/services/integration/tracking.service.ts

```
@@ -55,6 +55,7 @@
 }
 
   get tracking() {
+    console.log('got', (window as any).mixpanel);
```

ESLint

[error] Unexpected console statement.

(no-console)